src/core/addonManager.js

```diff
@@ -100,6 +102,28 @@ export function getAddon(
   return Promise.reject(new Error('Cannot check add-on status'));
 }
 
+export function hasAbuseReportPanelEnabled(
+  _mozAddonManager?: MozAddonManagerType = window.navigator.mozAddonManager,
+) {
+  if (_mozAddonManager || module.exports.hasAddonManager()) {
+    return _mozAddonManager.abuseReportPanelEnabled || false;
+  }
+  return false;
+}
+
+export function reportAbuse(
+  addonId: string,
+  { _mozAddonManager = window.navigator.mozAddonManager }: OptionalParams = {},
+) {
+  if (
+    (_mozAddonManager || module.exports.hasAddonManager()) &&
```

Comment:
Can we simply call `hasAbuseReportPanelEnabled()` by chance? The first part of the condition is already done in this function.

Reply:
Yes, good idea. I also don't see why we need to use `module.exports.hasAbuseReportPanelEnabled` as opposed to just `hasAbuseReportPanelEnabled`, so I've removed that.

Reply:
I think we need to keep `module.exports` because of the way we import this file in the rest of the project..

Reply:
Ah, okay

Reply:
When I put it back and just try to import the functions directly, I get an error about `module.exports is undefined`. I was thinking we didn't need to access these functions from the addonManager via `withInstallHelpers`, and could just import them from `addonManager.js` and call them, but that doesn't seem to work if I put back in `module.exports`. What exactly am I missing here? Is there a need to make it any more complicated than a simple import?

Reply:
That's because the `installAddon` HOC imports the `addonManager` code like this: https://github.com/mozilla/addons-frontend/blob/b8b13840ec63ae17d2d664acd26c64a14dadb4f6/src/core/installAddon.js#L38

Maybe we should fix that first?

Reply:
I'm not sure how that relates to what I'm doing. The installAddon HOC won't ever need access to these two new functions that are being added to `addonManager.js`, as they have nothing to do with installing add-ons. We could fix whatever the problem is with the HOC that was requiring `module.exports`, but is that actually related to this patch, or should that be addressed as a separate issue?

Reply:
> I'm not sure how that relates to what I'm doing. The installAddon HOC won't ever need access to these two new functions that are being added to `addonManager.js`, as they have nothing to do with installing add-ons. We could fix whatever the problem is with the HOC that was requiring `module.exports`, but is that actually related to this patch, or should that be addressed as a separate issue?

Well, the problem is that part of the added code relies on the existing code, which broke because it did not use `module.exports` a while ago. I get that it is not directly related to the issue but I am afraid that it will introduce a bug (the old bug was not caught by the test suite).

Reply:
Fair enough. I'm not sure exactly what I need to do to address the issue, though. Can you be more explicit?

Reply:
I believe we want to remove `module.exports` and make sure the rest of the code works as intended. I cannot retrieve the problem I was referring to before, though. I can't remember why `module.exports` is mandatory here...

Reply:
Re-thinking about it, if the two added functions work as expected without `module.exports`, even with a production build, then let's remove `module.exports` and let's move forward.

Reply:
Sounds good, thanks @willdurand